### PR TITLE
Added assumption for SC3-1-2-lang-valid

### PR DESCRIPTION
**Pull request by Brynanders**
Based on discussion in #348

### CODE REVIEW

**Comment by annethyme** [comment]
We discussed in #348 if the language attribute was applied to a single word, if that was a WCAG violation.

With the now added assumption, we cover both this case and also cases where a part of the content has been marked up with an incorrect language attribute, but where a language attribute is not needed at all because the language of the part is the same as the language of the whole page.

File: _rules/SC3-1-2-lang-valid.md [hunk 1]
```diff
@@ -27,7 +27,7 @@ The `lang` and `xml:lang` attributes have a [valid language subtag](#valid-langu
 
 ## Assumptions
 
-*There are currently no assumptions*
+The `lang` or `xml:lang` attribute is being used to comply to WCAG.
```
Comment (WilcoFiers) [inline _rules/SC3-1-2-lang-valid.md]: I don't think this explains anything. Can you explain?

Comment (Brynanders) [reply _rules/SC3-1-2-lang-valid.md]: @WilcoFiers fair point. It relates to this conversation and comment https://github.com/auto-wcag/auto-wcag/issues/348#issuecomment-456379859.

How about the following.

> This rule assumes that the presence of a `lang` or `xml:lang` attribute is being used to comply to WCAG. This rule doesn't test if the attribute is needed to comply to WCAG.

**Comment by annethyme** [comment]
@WilcoFiers, do you think an update like this has to go into CFC stage, or can we just merge as is?

**Comment by jkodu** [comment]
Discussed on the call, that we are happy to merge this.

**Pull request merged**